### ISSUE TITLE
Add other installation options

### ISSUE THREAD

**Issue by TechWiz-3** [TechWiz-3/happy-jar-cli#2]
Installation options for package managers such as `brew`, `apt` and `pacman`.

Currently I have no idea how to package and setup with these managers, any help much appreciated 🙏

**Comment by Karthik-PM** [comment]
can you assign me this issue

**Comment by TechWiz-3** [comment]
💯

**Comment by TechWiz-3** [comment]
Could you please let me know how you'll go about putting up the packages?

**Comment by TechWiz-3** [comment]
Any updates? @Karthik-PM

**Comment by Karthik-PM** [comment]
yeah i was thinking of compressing into a deb file and then creating a /usr/bin folder , but i was getting some issues
